examples/miscellaneous/plot_metadata_routing.py

```diff
@@ -154,27 +158,34 @@ def __init__(self, estimator):
     def get_metadata_routing(self):
         # This method defines the routing for this meta-estimator.
         # In order to do so, a `MetadataRouter` instance is created, and the
-        # right routing is added to it. More explanations follow.
+        # right routing is added to it. More explanations follow below.
         router = MetadataRouter(owner=self.__class__.__name__).add(
-            estimator=self.estimator, method_mapping="one-to-one"
+            estimator=self.estimator,
+            method_mapping=(  # <-- maybe use the other way with caller and callee,
+                # because it's more comprehensible without looking it up
+                "one-to-one"
+            ),
```

Comment:
Using the `.add(callee="fit", caller="fit")` etc. syntax here would not only be more understandable, it would also shorten this example, because it wouldn't need to be explained later in lines 305-307. And it would shorten all those long outputs of `print_routing(est)`.